lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -71,6 +71,23 @@ <h5 class="hd hd-5">${_("Rescore")}</h5>
 
     <br><br>
 
+    %if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS'):
+        <h5 class="hd hd-5">${_("Score Override")}</h5>
+        <label for="override-problem-score-single">${_("For the specified problem, override the learner's score.")}</label>
+        <br><br>
+        <label for="score-select-single">
+            ${_("New score for problem, out of the total points available for the problem *")}<br>
```

Comment:
The asterisk matches the other required fields on this page. Should we get rid of all of them?